app/split_integer.py

```diff
@@ -5,3 +5,6 @@ def split_integer(value: int, number_of_parts: int) -> list:
         parts.append(value // parts_left)
         value -= next_number
     return parts
+
+
+print(split_integer(5, 10))
```

Comment:
The print statement is used for debugging purposes. According to the checklist, you should remove any prints, comments, or debugging functions before committing your code.